docs/patterns/single-new-eks-observability-accelerators/single-new-eks-inferentia-neuron-opensource-observability.md

```diff
@@ -0,0 +1,268 @@
+# Single New EKS Cluster Open Source Observability Accelerator for Neuron-based clusters
```

Comment:
Rename this to `Single Cluster Open Source Observability - Neuron-based cluster`

Reply:
done